src/api/useApiUtil.tsx

```diff
@@ -1,5 +1,6 @@
 // useApiUtil.tsx
 import { useState, useEffect } from "react";
+import testData from "../test_data/testData.json"; // Make sure the path points to your test data file
```

Comment:
that only works at build time for a specific test data. I want us to have a bit more leeway on what data is returned. Why not make this a text field containing the JSON in the debug panel that we can set with some reasonable default samples?